doc/rst/usingchapel/executing.rst

```diff
@@ -385,6 +385,43 @@ This causes various software components, from launchers to the
 runtime, to be more considerate in how they use node resources.
 
 
+----------------
+Stack Traces
+----------------
+
+When compiled with unwind support (``CHPL_UNWIND=bundled`` or
+``CHPL_UNWIND=system``), Chapel can print a stacktrace when it halts due to an
+error. For example, when compiling and running the following program:
+
+   .. code-block:: chapel
+
+       // outofbounds.chpl
+       var A: [1..3] real;
+       A[5] = 1.0;
+
+   .. code-block:: sh
+
+        ./outofbounds
+
+   .. code-block:: sh
+
+        test/outofbounds.chpl:2: error: halt reached - array index out of bounds
+        note: index was 5 but array bounds are 1..3
+        Stacktrace
+
+        halt() at $CHPL_HOME/modules/standard/Errors.chpl:741
+        checkAccess() at $CHPL_HOME/modules/internal/ChapelArray.chpl:2675
+        chpl__init_outofbounds() at test/outofbounds.chpl:1
+
+The stacktrace will be printed. This behavior can also be disabled at runtime
+with the environment variable
```

Comment:
Cool feature that I did not know we had and these docs make sense to me -- the one thing that I would say is that the stack trace isn't shown in this above case when compiling with `--fast` (which disables the runtime checks), so that may be worth mentioning.

Reply:
Good point. This particular case with array out of bounds behaves extra differently because the check is removed entirely. But you're right that even `halt`s with `--fast` don't print the stacktrace, but they do print the halt. For example,

```
test/runtime/stacktrace/fact-stacktrace.chpl:2: error: halt reached
Stacktrace
# no stacktrace printed
```

Do you think changing the example to be an array bounds check (which is conflating the check and the halt) would be clearer? Or just explain that both things are going on here?